src/components/Nav.js

```diff
@@ -15,9 +16,9 @@ const _Nav = () => {
   const user = useContext(UserContext);
   return (
     <div className="flexnone">
-      <Navbar bg="light" variant="light" expand="md">
```

Comment:
have to leave expand="md" so it goes into a dropdown on mobile

Reply:
You're right! Will fix now, css change was for centring search bar.